subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -231,6 +284,15 @@ private LinePrefixingStyledTextOutput getLinePrefixingStyledTextOutput(FailureDe
 
             return new LinePrefixingStyledTextOutput(details.details, prefix, false);
         }
+
+        @Override
+        protected void uponFinishedVisiting() {
+            if (suppressedDuplicateBranchCount > 0) {
+                boolean plural = suppressedDuplicateBranchCount > 1;
+                LinePrefixingStyledTextOutput output = getLinePrefixingStyledTextOutput(failureDetails);
+                output.append(String.format("There were %d additional failure%s with the same cause that %s not printed.", suppressedDuplicateBranchCount, plural ? "s" : "", plural ? "were" : "was"));
+            }
```

Comment:
🤔 I would just do several branches here and not try to build up the string with a format string.